src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Custom/ExportClient/GrpcProtocolHelper.cs

```diff
@@ -0,0 +1,403 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Includes work from:
```

Comment:
I think that it will require us to extend LICENCSE.md/create rd-party notice and include it into nuget package. It is required by Apache license,

Reply:
We actually already have one: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/THIRD-PARTY-NOTICES.TXT

Both `LICENSE.txt` and `THIRD-PARTY-NOTICES.TXT` get included in the packages today. Ex: https://nuget.info/packages/OpenTelemetry.Exporter.OpenTelemetryProtocol/1.9.0